auth-web/src/models/pay/short-name.ts

```diff
@@ -63,6 +64,7 @@ export interface UnlinkedShortNameState {
   dateRangeReset: number
   clearFiltersTrigger: number
   selectedShortName: object
+  selectedAccount: object
```

Comment:
probably better defining the object, I saw some warnings complaining about the object type